components/CountryNavigator.js

```diff
@@ -0,0 +1,248 @@
+import { useEffect, useState, useMemo } from "react";
```

Comment:
🔎 There's some unused imports here, maybe we could add [prettier-plugin-organize-import](https://github.com/simonhaenisch/prettier-plugin-organize-imports) to do this for us automatically.